react-landing-page/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Card from "./components/Card";
+import Home from "./components/Home";
+import Nav from "./components/Nav";
+
+function App() {
+  return (
+    <div className="App">
+      <Nav 
```

Comment:
If there is nothing in between the opening and closing of the component tags; it is short to self close them after the props.

So <Nav props=...> </Nav> will become just <Nav props.../>